docs/docs/data-sources/Notion.md

```diff
@@ -0,0 +1,20 @@
+Notion data-sources can be added within Algomo by navigating to [**Algomo**](https://app.algomo.com/) > [**Data Sources**](https:app.algomo.com/data-sources) > [Create](https://app.algomo.com/data-sources/create/integations). Here you can add your notion document and then use it to answer questions or provide additional context for your bot's responses.
```

Comment:
This information is repeated in the steps below more clearly. I think just describing Notion data sources here would be enough. 

`Allow your bot to access your notion documentation with Notion Data Sources. This will create a snapshot of your notion pages, and save them within Algomo. `